optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/common/jaxrs/OptaShiftObjectMapperResolver.java

```diff
@@ -16,25 +16,28 @@
 
 package org.optaplanner.openshift.employeerostering.server.common.jaxrs;
 
-import javax.persistence.EntityNotFoundException;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
-import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.ext.ContextResolver;
-import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.optaplanner.persistence.jackson.api.OptaPlannerJacksonModule;
 
 @Provider
 public class OptaShiftObjectMapperResolver implements ContextResolver<ObjectMapper> {
 
     private final ObjectMapper objectMapper;
 
     public OptaShiftObjectMapperResolver() {
+        // JavaTimeModule defaults date format to ISO, so we don't need to deal with the headache of setting
+        // a universal date format that is ISO and includes an offset
         objectMapper = new ObjectMapper()
-                .registerModule(new JavaTimeModule());
+                .registerModule(OptaPlannerJacksonModule.createModule())
+                .registerModule(new JavaTimeModule())
+                .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+                .disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
```

Comment:
Let's add a comment above the first disable() call, such as:

// Write OffsetDateTime's (and similar) to JSON in the same format as OffsetDateTime.toString(), for example: 2017-01-01...(copy paste one from a json out)

Reply:
JavaTimeModule includes an OffsetDateTime serializer, which does not use toString()